plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -84,7 +82,6 @@ public function enqueue_scripts() {
 		if ( ! empty( $post_type_object->template ) ) {
 			$editor_settings['template']                 = $post_type_object->template;
 			$editor_settings['templateLock']             = ! empty( $post_type_object->template_lock ) ? $post_type_object->template_lock : false;
-			$editor_settings['__unstableResolvedAssets'] = $this->get_resolved_assets();
```

Comment:
Why is this being removed? Asking because I don't see an explanation in the PR description and, honestly, I have no idea what this code does 😆

Reply:
Aaah good question, I should of left a comment to clarify.
It appears that the `get_block_editor_settings` function that we trigger below with the editor settings triggers the `block_editor_settings_all` filter.
This in turn sets the `__unstableResolvedAssets` already: https://github.com/WordPress/gutenberg/blob/trunk/lib/compat/wordpress-6.3/script-loader.php#L105

Reply:
The above links to a 6.3 script loader script, but it seemed to also be loaded for me in 6.2